docs/ja/guide/README.md

```diff
@@ -0,0 +1 @@
+../../README.ja.md
```

Comment:
Is this file translated??

Reply:
Yes. It looks like it correctly updated the symlink to the new path when I run it locally.